jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStepTest.java

```diff
@@ -297,6 +300,25 @@ public void testGetCachedBaseImages_emptyCache()
     Assert.assertEquals(Arrays.asList(), pullBaseImageStep.getCachedBaseImages());
   }
 
+  @Test
+  public void testGetCachedBaseImages_cachedWithoutAllLayers()
```

Comment:
What do you think of this? 

```suggestion
  public void testGetCachedBaseImages_partiallyCached_emptyListReturned()
```

Reply:
Makes sense! Updated